Destination/RotatableStreamDestination.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Destination;
+
+use Psr\Log\LoggerInterface;
+use Zenstruck\BackupBundle\Rotator\Rotator;
+
+class RotatableStreamDestination extends StreamDestination
```

Comment:
Ok, here it is, no new interface, no abstract class, only subclass of StreamDestination